d3rlpy/logging/wandb_adapter.py

```diff
@@ -52,6 +58,21 @@ def close(self) -> None:
         """Closes the logger and finishes the WandB run."""
         self.run.finish()
 
+    def watch_model(
+        self,
+        epoch: int,
+        step: int,
+        logging_steps: Optional[int],
+        algo: TorchModuleProtocol,
+    ) -> None:
+        if not self._is_model_watched:
+            self.run.watch(
+                tuple(algo.impl.modules.get_torch_modules().values()),
+                log="gradients",
+                log_freq=logging_steps,
+            )
+            self._is_model_watched = True
```

Comment:
Instead of `watch`, we can use 
```python
self.run.log({"name": wandb.Histogram(...) })
```

not sure which direction I should choose

Reply:
For now, I'm okay with the either way. The current implementation also looks good to me.